DfE.FindInformationAcademiesTrusts/Pages/Trusts/Contacts.cshtml.cs

```diff
@@ -1,10 +1,36 @@
 using DfE.FindInformationAcademiesTrusts.Data;
+using Microsoft.AspNetCore.Mvc;
 
 namespace DfE.FindInformationAcademiesTrusts.Pages.Trusts;
 
 public class ContactsModel : TrustsAreaModel
 {
+    public Governor? ChairOfTrustees { get; set; }
+
+    public Governor? AccountingOfficer { get; set; }
+
+    public Governor? ChiefFinancialOfficer { get; set; }
+
+    public string ContactNameNotAvailableMessage = "No contact name available";
+
+    public string ContactEmailNotAvailableMessage = "No contact email available";
+
+    public string ContactInformationNotAvailableMessage = "No contact information available";
+
     public ContactsModel(ITrustProvider trustProvider) : base(trustProvider, "Contacts")
     {
     }
+
+    public override async Task<IActionResult> OnGetAsync()
+    {
+        var pageResult = await base.OnGetAsync();
+        ChairOfTrustees = Trust.Governors.FirstOrDefault(x =>
+            x.Role == "Chair of Trustees" && (x.DateOfTermEnd == null || x.DateOfTermEnd >= DateTime.Now));
```

Comment:
`(x.DateOfTermEnd == null || x.DateOfTermEnd >= DateTime.Now)` would be a good property to have on the `Governor` type because it will be used in other places as well as being repeated here